content/reference/admin/private_locations/configuration/dedicated/index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Dedicated Machines"
+description: "Deploy Load Generators on dedicated machines that you manage."
+lead: "Deploy Load Generators on dedicated machines that you manage."
+date: 2023-01-12T16:46:04+00:00
+lastmod: 2023-10-13T08:10:39+00:00
+weight: 22055
+---
+
+## Dedicated Machines
+
+To use Dedicated Machines private locations, the control plane must have access to your dedicated machines.
+
+To set up a dedicated location configuration, ensure that each host is reachable through SSH on a specific port.
+The control plane possesses a private key for establishing connections, while the corresponding public key is shared and configured on every host in the location.
+
+### Host prerequisites
+
+Location hosts environment must include:
+- Java runtime environment
+- `bash` the GNU Project's shell
+- `jq` a lightweight and flexible command-line JSON processor
+- `curl` command line tool for transferring data with URLs
+- `~/.ssh/authorized_keys` with control plane public key
+
+### Control plane configuration file
+
+```bash
+control-plane {
+  # Control plane token
+  token = "cpt_example_c7oze5djp3u14a5xqjanh..."
+  # Control plane token with an environment variable
+  token = ${?CONTROL_PLANE_TOKEN}
+  # Control plane description (optional)
+  description = "my control plane description"
+  # Locations configurations
+  locations = [
+    {
+      # Private location ID, must be prefixed by prl_, only consist of numbers 0-9, 
+      # lowercase letters a-z, and underscores, with a max length of 30 characters
+      id = "prl_private_location_example"
+      # Private location description (optional)
+      description = "Private Location on Dedicated Machines"
+      # Private location type
+      type = "dedicated"
+      ssh {
+        # SSH user
+        user = "gatling"
+        ## SSH private key
+        private-key {
+          # SSH private key path used to secure connections
+          path = "private/key/path"
+          ## SSH private key password (optional, default: none)
+          password = "password"
+           # SSH private key password with an environment variable
+          password = ${?PRIVATE_KEY_PASSWORD}
+        }
+        # SSH port (optional, default: 22)
+        # port = 22
+        # SSH connection timeout (optional, default 15 seconds)
+        # connection-timeout = 15 seconds
```

Comment:
Should we have a convention that comment on commented code should be commented? :thinking: 

```suggestion
        # # SSH port (optional, default: 22)
        # port = 22
        # # SSH connection timeout (optional, default 15 seconds)
        # connection-timeout = 15 seconds
```

or:

```suggestion
        ## SSH port (optional, default: 22)
        # port = 22
        ## SSH connection timeout (optional, default 15 seconds)
        # connection-timeout = 15 seconds
```